### PR TITLE
Bower package converted to a Rubygem

### PR DESCRIPTION
**Pull request by skateman**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 node_modules/
 bower_components/
+/Gemfile.lock
```
Comment (abonas) [inline .gitignore]: is this needed due to building it locally?

Comment (skateman) [inline .gitignore]: yes, like `pkg`

**Comment by stefwalter** [comment]
Why are we packaging javascript as ruby? This component has nothing to do with ruby.

**Comment by skateman** [comment]
@stefwalter we (miq) need a package from this which is compatible with the rails' asset-pipeline and also not collides with the `angularjs-rails` rubygem.

**Comment by stefwalter** [comment]
It seems very odd to push such project specific oddities upstream. But if it needs to be done like this, then:
1. Please put all of this in a subdirectory called `contrib/gem`
2. Please don't copy and paste any information. The version number, dependencies, should be updated in exactly one place. In particular this information is currently being committed in these files:
   - `lib/kubernetes-topology-graph/version.rb`
   - `kubernetes-topology-graph.gemspec`

**Comment by stefwalter** [comment]
Actually this seems rails specific, not even plain ruby specific. So the subdirectory would should probably be called `contrib/rails`

**Comment by skateman** [comment]
@stefwalter the version is copied by a rake task from the `bower.json` file!

**Comment by stefwalter** [comment]
> the version is copied by a rake task from the bower.json file!

I guess then don't commit it.

**Comment by skateman** [comment]
okay, give me some time to move the stuff :)

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'rake'
+require 'json'
+require 'bundler/gem_tasks'
+
+desc 'Copy version number from the JSON file'
+task :generate do
+  path = 'lib/kubernetes-topology-graph/version.rb'
+  version = JSON.parse(File.read('bower.json'))['version']
+  content = File.read(path).sub(/VERSION\s*=\s*['"][0-9\.]+['"]/, "VERSION = '#{version}'")
+  File.open(path, 'w') { |f| f.write(content) }
```
Comment (Fryguy) [inline Rakefile]: Can be simplified to `File.write(path, content)`

File: assets/javascripts/kubernetes-topology-graph.js [hunk 3]
```diff
@@ -0,0 +1 @@
+../../dist/topology-graph.js
```
Comment (Fryguy) [inline assets/javascripts/kubernetes-topology-graph.js]: Have you done this before?  Do symlinks work properly with the asset pipeline?  Just curious as I've never tried it myself.

Comment (skateman) [inline assets/javascripts/kubernetes-topology-graph.js]: @Fryguy tested, yes!

**Comment by skateman** [comment]
@stefwalter fixed, review pls!

**Comment by stefwalter** [comment]
@Fryguy In a mailing list thread you were saying this pull request should no longer be necessary. Am I imagining that? Did I misunderstand?

**Comment by stefwalter** [comment]
I'm told this is no longer necessary. So I'll close it. Hope I got that right.

**Pull request closed without merging**